cmd/lotus-worker/main.go

```diff
@@ -120,7 +120,12 @@ func main() {
 var stopCmd = &cli.Command{
 	Name:  "stop",
 	Usage: "Stop a running lotus worker",
-	Flags: []cli.Flag{},
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:  "forcefully",
```

Comment:
I would prefer to name this `--force`

Reply:
Yeah, that seems better! Updated in https://github.com/filecoin-project/lotus/pull/11193/commits/f2ca2e01b9f5479810e1ba72eff9cd6b50ac939a